molecule/broker-scale-up/converge.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Add new brokers Nodes into in-memory inventory to scale-up
+  hosts: all
+  gather_facts: false
+  tasks:
+    - name: "Add new Broker Nodes"
+      add_host:
+        hostname: kafka-broker4
+        ansible_connection: community.docker.docker
+        groups: kafka_broker
+        tag: kafka_broker
+
+    - name: "Add new Broker Nodes"
+      add_host:
+        hostname: kafka-broker5
+        ansible_connection: community.docker.docker
+        groups: kafka_broker
+        tag: kafka_broker
+
+    - name: "Print Test"
```

Comment:
Please remove the debug tasks. 
You can have similar assertions in the verify phase.